src/platform/site-wide/sass/minimal.scss

```diff
@@ -21,11 +21,6 @@
 @import "~@department-of-veterans-affairs/formation/sass/formation-overrides/foundation-sites";
 // @import "~foundation-sites/scss/foundation/components/block-grid";
 
-@import "~@fortawesome/fontawesome-free/scss/fontawesome";
```

Comment:
How can we be sure these aren't used?

Reply:
DST has been systematically swapping any instances of  `fa-...` on `vets-website` also looking at  scss rules that use font awesome and in both cases swapping for analogous va-icons.

I admit removing the import is a bold step,  but as we technically are  dropping support for FontAwesome in favor of USWDS icons, it is a logical conclusion.

Reply:
Would these imports impact any application in vets-website? For example: https://github.com/department-of-veterans-affairs/vets-website/blob/546bb5b2c1270a1128cbcb402519e9e645c65e24/src/applications/my-education-benefits/sass/my-education-benefits.scss#L258

Reply:
There a odd cases, as  in the Crisis Modal where the icon is an inline svg. I will have to check about what to to in that instance. But the thing is, our goal is NOT to use Font Awesome in vets-website. So if it's an instance of `fa` in `vets-website` we should remove it and swap in `va-icon`.